app/src/main/java/com/github/odaridavid/weatherapp/data/weather/local/converters/Converters.kt

```diff
@@ -0,0 +1,58 @@
+package com.github.odaridavid.weatherapp.data.weather.local.converters
+
+import androidx.room.ProvidedTypeConverter
+import androidx.room.TypeConverter
+import com.github.odaridavid.weatherapp.data.weather.local.entity.DailyWeatherEntity
+import com.github.odaridavid.weatherapp.data.weather.local.entity.HourlyWeatherEntity
+import com.github.odaridavid.weatherapp.data.weather.local.entity.TemperatureEntity
+import com.github.odaridavid.weatherapp.data.weather.local.entity.WeatherInfoResponseEntity
+import com.google.gson.Gson
+import com.google.gson.reflect.TypeToken
+
+@ProvidedTypeConverter
+class Converters {
+    private val gson = Gson()
+
+    @TypeConverter
+    fun fromHourlyWeatherEntityList(hourlyList: List<HourlyWeatherEntity>): String {
```

Comment:
Just curious

Why do you want to save a list of entities as a string - is it not better to basically use relationships?

Also, avoid gson as suggested earlier

Use typeconverters only when converting data types not supported by room natively

Reply:
My bad. Relationships would be a better approach. Let me make the adjustments and remove gson.